aggregator/.env.local.example

```diff
@@ -6,8 +6,8 @@ ORIGIN=http://localhost:3000
 PORT=3000
 
 NETWORK_CONFIG_PATH=../contracts/networks/local.json
-PRIVATE_KEY_AGG=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
-PRIVATE_KEY_ADMIN=0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
+PRIVATE_KEY_AGG=0x6ed6982f65033352b8c6a821a00d462b3774290569fde98331cd828da7ccf6fb
+PRIVATE_KEY_ADMIN=0xafb3afac09efb080cb5f925ba956af68146804e4b892e5060b8c6e96b1ecdf77
```

Comment:
FYI @voltrevo @kautukkundan @JohnGuilding @blakecduncan it looks like the local hardhat network is now pulling accounts from `hardhat.config.ts`, where we use `MAIN_MNEMONIC` from the `.env` file in contracts rather than the pre-loaded ones it used to.

Reply:
Turn out ^ only applies locally, in CI I'm still seeing the old values be needed. Unsure why.